.ci/idea-format.bat

```diff
@@ -0,0 +1,55 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: IntelliJ IDEA inspections for checkstyle.
+::
+:: Example:
+:: SET IDEA_PATH=C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.1\bin\idea.bat
+:: .ci\idea-inspection.bat
+::----------------------------------------------------------------------
```

Comment:
```suggestion
: '
This is a
very neat comment
in bash

----------------------------------------------------------------------
 IntelliJ IDEA inspections for checkstyle.

:: Example:
 SET IDEA_PATH=C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.1\bin\idea.bat
 .ci\idea-inspection.bat
----------------------------------------------------------------------
'
```
maybe use multi line https://stackoverflow.com/questions/43158140/way-to-create-multiline-comments-in-bash

or at least consistent `#` or `::`

idk why mixed.